packages/core/redux/bitcoin/thunks.js

```diff
@@ -23,6 +23,10 @@ import { pendingEntrySelector, getBitcoinSigner } from './selectors'
 import { notify, notifyError } from '../notifier/actions'
 import BitcoinMiddlewareService from './BitcoinMiddlewareService'
 
+import { getAddressUTXOS } from '../bitcoin-like-blockchain/thunks'
```

Comment:
It is enough to have only 'export' here (at line 28). `getAddressUTXOS` will be available to use in this file.

Reply:
No, it is not enough. Without `import { getAddressUTXOS } from '../bitcoin-like-blockchain/thunks'` the `getAddressUTXOS` function will be available in the files importing `packages/core/redux/bitcoin/thunks.js` but not in the `packages/core/redux/bitcoin/thunks.js` (where the `getAddressUTXOS` function is also used)

Reply:
Yep. My bad.